src/components/dialogs/limits/limits-pane-utils.ts

```diff
@@ -220,6 +225,21 @@ export const updateTemporaryLimits = (
     return updatedTemporaryLimits;
 };
 
+export const mapServerOLGtoOperationalLimitsGroupsFormInfos = (currentLimits: CurrentLimits[]) => {
```

Comment:
well for the naming, there is OLG and OperationalLimitsGroups in the method name